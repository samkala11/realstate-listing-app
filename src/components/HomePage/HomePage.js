import React from 'react';

const Home = () => {
  return (
    <div data-testid="home-page-title" className="Home">
      Home
    </div>
  )
}

export default Home;