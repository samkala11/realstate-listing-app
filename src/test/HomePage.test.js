import { render, screen, cleanup } from '@testing-library/react';
import HomePage from '../components/HomePage/HomePage';
import renderer from 'react-test-renderer';

afterEach(cleanup);

test('Renders Home page successfully', () => {
    render(<HomePage />);
    const homePageTitle = screen.getByTestId('home-page-title');

    expect(homePageTitle).toBeInTheDocument();
    expect(homePageTitle).toHaveTextContent('Home')
  
});

test('Home page matches snapshot', () => {
    const tree = renderer.create(<HomePage />).toJSON();
    expect(tree).toMatchSnapshot();
})