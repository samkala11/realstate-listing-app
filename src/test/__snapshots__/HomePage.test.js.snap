// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<div
  className="Home"
  data-testid="home-page-title"
>
  Home
</div>
`;
